titus-testkit/src/main/java/com/netflix/titus/testkit/perf/load/plan/generator/ConstantLoadJobExecutableGenerator.java

```diff
@@ -16,42 +16,35 @@
 
 package com.netflix.titus.testkit.perf.load.plan.generator;
 
-import com.google.common.base.Preconditions;
 import com.netflix.titus.api.jobmanager.model.job.JobDescriptor;
 import com.netflix.titus.testkit.perf.load.plan.ExecutionPlan;
 import com.netflix.titus.testkit.perf.load.plan.JobExecutableGenerator;
-import rx.Observable;
-import rx.Subscriber;
-import rx.observers.SerializedSubscriber;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.FluxProcessor;
+import reactor.core.publisher.UnicastProcessor;
 
 public class ConstantLoadJobExecutableGenerator extends JobExecutableGenerator {
 
     private final Executable executable;
-    private final int numberOfJobs;
-
-    private volatile Subscriber<? super Executable> scenarioSubscriber;
+    private final FluxProcessor<Executable, Executable> executionPlans;
 
     public ConstantLoadJobExecutableGenerator(String owner, JobDescriptor<?> jobSpec, ExecutionPlan plan, int numberOfJobs) {
         this.executable = new Executable(owner, jobSpec, plan);
-        this.numberOfJobs = numberOfJobs;
+        this.executionPlans = UnicastProcessor.<Executable>create().serialize();
```

Comment:
Will this work for multiple subscriptions?

Reply:
the original code was trying to force a single subscription